frontend/app/src/components/home/profile/SendCrypto.svelte

```diff
@@ -4,40 +4,42 @@
     import { Cryptocurrency, OpenChat, cryptoLookup } from "openchat-client";
     import Input from "../../Input.svelte";
     import { _ } from "svelte-i18n";
-    import Send from "svelte-material-icons/Send.svelte";
+    import QrcodeScan from "svelte-material-icons/QrcodeScan.svelte";
     import { logger } from "../../../utils/logging";
     import { toastStore } from "../../../stores/toast";
     import { iconSize } from "../../../stores/iconSize";
     import Scanner from "./Scanner.svelte";
 
     export let token: Cryptocurrency;
     export let amountToSendE8s: bigint;
-    export let scanningUnsupported: Boolean;
+    export let sending = false;
+    export let valid = false;
 
     const client = getContext<OpenChat>("client");
     const user = client.user;
     const dispatch = createEventDispatcher();
 
     let validAmount = false;
     let targetAccount: string = "";
-    let sending = false;
     let scanner: Scanner;
 
     $: account = token === "icp" ? user.cryptoAccount : user.userId;
     $: cryptoBalance = client.cryptoBalance;
     $: transferFees = cryptoLookup[token].transferFeesE8s;
-    $: valid =
-        validAmount &&
-        amountToSendE8s > BigInt(0) &&
-        targetAccount !== "" &&
-        targetAccount !== account;
     $: symbol = cryptoLookup[token].symbol;
+    $: {
```

Comment:
Why this change out of interest?

Reply:
because it makes it possible to two-way bind it to the parent component which is now necessary since send button is up a level now and should only be enabled when valid is true.